runtime/lua/vim/filetype.lua

```diff
@@ -834,6 +901,16 @@ local extension = {
   ctp = 'php',
   php = 'php',
   phpt = 'php',
+  php0 = 'php',
+  php1 = 'php',
```

Comment:
`php0` and `php1` aren't a thing.

Reply:
Gotta tell that to Vim; they recognize this and so will we.